definitions/checks/disk/available_space_var.rb

```diff
@@ -0,0 +1,26 @@
+module Checks
+  module Disk
+    class AvailableSpaceCandlepin < ForemanMaintain::Check
+      metadata do
+        label :available_space_cp
+        description 'Check to make sure /var/lib/candlepin has enough space'
+        tags :pre_upgrade
+        confine do
+          feature(:candlepin) && feature(:candlepin).current_minor_version == '3.1'
```

Comment:
If we want to restrict this for only 3.1 version then it is fine, but on the candlepin BZ - https://bugzilla.redhat.com/show_bug.cgi?id=1898605 I see that target milestone is not set. So possibly we can do. 
```suggestion
          feature(:candlepin) && check_min_version('candlepin', '3.1')'
```